hawc/apps/assessment/templates/assessment/assessment_downloads.html

```diff
@@ -6,91 +6,156 @@ <h2>{{object}} downloads</h2>
   <p class='form-text text-muted'>
     All data from HAWC are exportable into Excel. Developer exports in JSON
     format are also available (please <a target="_blank" href="{% url 'contact' %}">contact us</a> for more information).
+    Visualizations can be exported into SVG and PNG formats;
+    you can download each individual visualization when being viewed.
   </p>
-
-  <ul>
+  <div class="card-columns">
     {% if assessment.enable_literature_review and assessment.has_lit_data %}
```

Comment:
@casey1173 one thought that came out of our meeting was to show the cards even if there's no data, so users know there's no data for this domain. So I think this would be moving the if login inside the card-body, to basically show either the body with buttons or a "no data available" body